qutip/core/coefficient.py

```diff
@@ -353,6 +367,9 @@ def coeff_from_str(base, args, args_ctypes, compile_opt=None):
     return coeff(base, keys, const, args)
 
 
+coefficient_builders[str] = coeff_from_str
```

Comment:
Could we move this one with the rest for clarity?

Reply:
Easier said than done, we would need to split the file in 2...
We must add it after defining `coeff_from_str` thus we can't add it with the other, but building `coefficient_builders` too far from the function would be confusing.

Reply:
I see, thanks for clarifying it!